packages/cozy-client/src/types.js

```diff
@@ -156,12 +156,28 @@ import { QueryDefinition } from './queries/dsl'
  * @property {object} [cozyMetadata] - Cozy Metadata
  */
 
+/**
+ * Metadata on io.cozy.files documents
+ *
+ * This is a first step, to continue
+ *
+ * Filled according to:
+ * https://docs.cozy.io/en/cozy-doctypes/docs/io.cozy.files_metadata/
+ * https://docs.cozy.io/en/cozy-doctypes/docs/io.cozy.notes/
+ *
+ * @typedef {object} FileMetadata - A io.cozy.files document's metadata
+ * @property {string} [content] - Content of a note
```

Comment:
I've created https://github.com/cozy/cozy-doctypes/pull/208/files with more accurate type. 

content is an object. ;)